src/main/js/dashboard-container.js

```diff
@@ -6,9 +6,27 @@
 
 import React from "react";
 import Dashboard from "./dashboard";
+import {createRoot} from "react-dom/client";
+import AccountService from "./account/account-service";
+import Account from "./account/account";
 
 const DashboardContainer = (props) => {
-  return <Dashboard />;
+    let service = new AccountService();
+
+    service.findAllBankAccount()
+        .then(function (response) {
+            createRoot(document.getElementById("dashboardAccounts"))
+                .render(<Account bankAccount={response.data.additionalData}/>);
```

Comment:
Well that's not the React way of doing this. You're supposed to render elements inside the Dashboard component. You never have to raw DOM manipulation or call render. Take a look at my components and also some React fundamentals.

Reply:
Should I do like this `ReactDOM.render(<Account bankAccount={response.data.additionalData}/>, document.getElementById("dashboardAccounts"));`

Reply:
Nope. Each component simply returns other components. Create small reusable components which are returned the container component. Take another look at React fundamentals it's worth it to build that base understanding.

`return <Login handleLoginSubmit={handleLoginSubmit} />;`

Reply:
<img width="1714" alt="Screen Shot 2022-06-03 at 8 38 23 PM" src="https://user-images.githubusercontent.com/105374562/171969625-b56cc69c-6041-4908-a464-2c0dfd80ebf4.png">
You page is breaking in dev.

Reply:
![image](https://user-images.githubusercontent.com/53858277/171970205-a8fd5357-75ff-4274-a66d-5a91a35a8ac1.png)

Reply:
I have normal here

Reply:
Have done and commited